prow/config/external_plugins_config.yaml

```diff
@@ -1097,10 +1097,44 @@ ti-community-tars:
       Your PR was out of date, I have automatically updated it for you.
 
 ti-community-label-blocker:
+  - repos:
+      - pingcap/tidb
+    block_labels:
+      - regex: "^status/LGT[\\d]+$"
+        actions:
+          - labeled
+          - unlabeled
+        trusted_users:
+          - ti-chi-bot
+      - regex: "^status/can-merge$"
+        actions:
+          - labeled
+          - unlabeled
+        trusted_users:
+          - ti-chi-bot
+      - regex: "^cherry-pick-approved$"
+        actions:
+          - labeled
+          - unlabeled
+        trusted_teams:
+          - qa-release-merge
+        trusted_users:
+          - ti-chi-bot
+      - regex: "^do-not-merge/blocked-paths$"
+        actions:
+          - labeled
+          - unlabeled
+        trusted_users:
+          - qw4990
+          - bb7133
+          - XuHuaiyu
+          - cfzjywxk
+          - morgo
+          - ti-chi-bot
```

Comment:
is it possible a random user remove the do-not-merge label by `/unlabel do-not-merge/blocked-paths`?

Reply:
No, because do-not-merge type labels are not in the trusted list, the robot will not respond.

https://github.com/ti-community-infra/configs/blob/b7c9df395a9819df91660364cdf0f518e5086e3c/prow/config/external_plugins_config.yaml#L423